### ISSUE TITLE
Css modules

### ISSUE THREAD

**Issue by glebmachine** [bline/jade-html-loader#18]
Hi!)

Can you add `css-modules` support please.
https://github.com/css-modules/css-modules
fr: https://github.com/css-modules/webpack-demo

**Comment by dwayne** [comment]
@glebmachine What does CSS modules have to do with Jade?

**Comment by glebmachine** [comment]
CSS modules export object like:

``` js
{
  .className: unique_hash_for_this_classname1,
  .className2: unique_hash_for_this_classname2,
}
```

All css selectors become local without BEM.

**Comment by dwayne** [comment]
Okay. So do you want the template function to be called with this object (the locals) that CSS modules exports?

**Comment by glebmachine** [comment]
Hm, is it possible to use require function right into template, like in the jade-loader or plain node js?

**Comment by dwayne** [comment]
From my understanding, `jade-loader` will return the template function for you. That template function takes a locals object that it uses to substitute into the template when executed. Maybe try using that. See [here](https://github.com/webpack/jade-loader/blob/d9caf449dad6458ebc8d970cad330394cfdac424/index.js#L170).

**Comment by glebmachine** [comment]
Yes, they provide global variables into template scope, so, require become available from template scope.

``` jade
var css = require('style.css')
div(class=css.className1)
vid(class=css.className2)
```
